ts/nni_manager/training_service/reusable/environments/openPaiEnvironmentService.ts

```diff
@@ -182,7 +182,7 @@ export class OpenPaiEnvironmentService extends EnvironmentService {
         let environmentRoot: string;
         if (environment.useSharedStorage) {
             environmentRoot = component.get<SharedStorageService>(SharedStorageService).remoteWorkingRoot;
-            environment.command = `${component.get<SharedStorageService>(SharedStorageService).remoteMountCommand} && cd ${environmentRoot} && ${environment.command}`;
+            environment.command = `${component.get<SharedStorageService>(SharedStorageService).remoteMountCommand.replace(/echo -e /g, `echo `).replace(/echo /g, `echo -e `)} && cd ${environmentRoot} && ${environment.command}`;
```

Comment:
What is this doing?

Reply:
Change all `echo` to `echo -e`. Because in pai we need `-e` to escape character, but in other training services we don't need. Seems if we run the command by `python` or `ts`, we don't need `-e`, but in bash, we need `-e`. I am not sure, but it runs well under this.

Reply:
So why it's replaced twice?

Reply:
To avoid the situation of `echo` and `echo -e` exist at the same time.